clients/admin-ui/src/types/api/models/SupportedLanguage.ts

```diff
@@ -16,6 +16,7 @@ export enum SupportedLanguage {
   EL = "el",
   EN = "en",
   ES = "es",
+  ES_MX = "es-MX",
```

Comment:
was this just something we missed before?

Reply:
I think there was some uncertainty with whether we would support Mexican Spanish in addition to Castillian, but I guess that got resolved?  Not 100% sure, to be honest.

Reply:
This was added later, we originally only had "es" (and we saving mexican spanish translations underneath them)! And then we received castillian translations so we switched them and those were saved under es, and we added new "es-mx" for Spanish (Mexico)!